components/scream/src/physics/spa/spa_functions_impl.hpp

```diff
@@ -334,113 +333,18 @@ template <typename S, typename D>
 void SPAFunctions<S,D>
 ::get_remap_weights_from_file(
```

Comment:
And, tbc, this routine is all run once and only at model init?

Reply:
Yes, we only gather remap weights once.

Reply:
als0 als0 doc